misc/test_bit_struct.py

```diff
@@ -42,9 +42,14 @@ def test_simple_singleton():
 
     @ti.kernel
     def foo():
-        print(x[None])
+        x[None] = 2**13-1
+        print('x: (2**13-1)', x[None])
```

Comment:
You can `ti.init(debug=True)` and then use `assert` in Taichi kernels. Then you don't need to eyeballing the outputs anymore.